### ISSUE TITLE
Explaining the numbers to innumerate folks

### ISSUE THREAD

**Issue by kdpenner** [appliedbinf/covid19-event-risk-planner#37]
I've been posting time series plots of risk for the D.C. metro area (D.C. + northern VA + parts of MD). I get lots of questions. Most of them are variants of the following:

> Me: In a gathering of 10 people, there’s a 15–20% probability that at least 1 person is infected.

> Them: Wouldn’t that mean that 10% of the total population currently has a COVID infection?

To which my response is:

> No.
> If the probability is 100%, one or more people are infected---there is no doubt. If the probability is 0%, no person is infected---there is no doubt. If the probability is 20%, you are more certain that one or more people are infected than if the probability is 10%.

I get so many questions like this that I'm no longer going to use percentages.

Sometimes there's a follow-up comment along the lines of:

> Them: That doesn't help me, because I don't know where to set the risk I'm willing to personally accept.

To which my response is:

> Me: Say you're willing to accept a risk of 0.17. Flip a coin 10 times. The risk of getting 7 or more heads is 0.17.

A better second response has been hard for me to come up with, but a coin flip is something people know.

Food for thought.

**Comment by carkar7** [comment]
I’m trying to put myself into the mindset of “Them” when they say:

“Wouldn’t that mean that 10% of the total population currently has a COVID infection?”

But I can’t understand how they are drawing this conclusion based on the information that “In a gathering of 10 people, there’s a 15–20% probability that at least 1 person is infected.”

To me, it would make more sense for them to ask:

“Wouldn’t that mean that 15-20% of the total population currently has a COVID infection?“ just based on my assumption of how a person would misinterpret the numbers.

Regardless. For your follow up response, what if you expanded on the first response by saying something like:

“If there were 100 families of 10 people who got together for thanksgiving, of those 100 families, 15 to 20 of them will have at least one person who is positive for COVID”

**Comment by kdpenner** [comment]
> To me, it would make more sense for them to ask:
“Wouldn’t that mean that 15-20% of the total population currently has a COVID infection?“ just based on my assumption of how a person would misinterpret the numbers.

That's what I think, too, but it's possible the 10% quote was a typo or them forgetting that the number is 20 after scrolling down a long thread. I didn't ask. I've seen the mistake often enough that I made the correction and moved on.

> For your follow up response, what if you expanded on the first response by saying something like:
“If there were 100 families of 10 people who got together for thanksgiving, of those 100 families, 15 to 20 of them will have at least one person who is positive for COVID”

Valid enough. The intuition I developed as a TA tells me that most folks would dismiss the threat if presented this way, thinking it's a binary "I'm either in that group or not". My interactions with questioners are over the internet, so my intuition could be wrong.
